.github/workflows/gen_alpine3.15.yml

```diff
@@ -56,12 +56,7 @@ jobs:
         with:
           submodules: "recursive"
       - name: "Install Rust"
-        uses: actions-rs/toolchain@v1
-        with:
-          profile: "minimal"
-          toolchain: "stable"
-          override: true
-          components: "rustfmt"
+        uses: dtolnay/rust-toolchain@stable
         env:
           ACTIONS_ALLOW_UNSECURE_COMMANDS: "true"
```

Comment:
Do you know if this is still needed with this action? Can you try removing it?

Reply:
I have no idea, but I saw it being used without it, I will push a fix and you can try to run the ci.

Reply:
Seeing that CentOS failed to run curl, it might not be as simpler as removing ALLOW_UNSAFE_COMMANDS, I will recheck the actions used by dtonlay and see how he does it.

Reply:
Seeing that the rest of the builds have succeeded I think you are right.

Would you like me to report the issue with CentOS 7 upstream?

If I remember you were planning on dropping the CI for some Linux OSs, so I could also wait for that.

Reply:
I have users running on centos7, so I can't just drop support for that build.
Realistically, centos7 is not going to update the version of curl just for that command line flag, as it is essentially frozen except for critical security issues until it reaches its EOL date.

I think another option, that is honestly quite reasonable, is that we just manually run the rustup installer for centos7 in our centos7 workflow.  It's not much more than a one-liner anyway.

Reply:
Do you mean to modify the `yml` file or the python script? I am assuming it's the `yml` file.

Reply:
I patched the python file to generate the CI, now it manually downloads and installs rustup.